test/java/seedu/addressbook/logic/LogicTest.java

```diff
@@ -201,6 +201,30 @@ public void execute_list_showsAllPersons() throws Exception {
                               expectedList);
     }
 
+    @Test
+    public void execute_sort_showsAllPerson() throws Exception {
+        // prepare expectations
+        TestDataHelper helper = new TestDataHelper();
+        Person person1 = helper.generatePersonWithName("John Tan");
```

Comment:
Might want to name these variables `John`, `Ali`, etc instead of `person1` `person2` so as to make the generation of `expectedList` and `unorderedList` more readable.